plugins/lookup/vmware_moid.py

```diff
@@ -0,0 +1,526 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+lookup: vmware_moid
```

Comment:
I think we should name this as `moid` or `get_moid` or something similar. With the collection, `vmware_` is redundant in the name.

Reply:
Thank you for reviewing @Akasurde. I called it get_moid